docs/section5/Rule5-31.md

```diff
@@ -0,0 +1,46 @@
+
+# Envelope - Rule 5-31  
+
```

Comment:
Rule numbering is a bit confusing since the title is for Rule 5-26 but it's actually rule 5-31

Reply:
PR rule numbering is now changed to 5-31.

Reply:
PR rule numbering is now changed to 5-31.